sample-notificator-domain/src/main/groovy/life/qbic/business/notification/refactor/FailureEmail.java

```diff
@@ -0,0 +1,7 @@
+package life.qbic.business.notification.refactor;
+
+/**
+ * <b>An email informing about system failure.</b>
+ */
+public interface FailureEmail {
```

Comment:
This is an empty class?

Reply:
Yes this interface is used to achieve type boxing.

Reply:
You were right, it turns out I introduced it and forgot to remove it as it was not used in the end.